CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+### citus v11.0.5 (August 1, 2022) ###
+
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
+
+* Fixes a concurrency bug between creating co-located distributed table and
```

Comment:
```suggestion
* Fixes a concurrency bug between creating co-located distributed table and

* Fixes a crash that can happen due to catalog read in shmem_exit
```

Reply:
This suggestion is still missing some parts I guess. The first entry ends with the word `and`

Reply:
See https://github.com/citusdata/citus/pull/6108/files#r934247807